pkgs/servers/mail/sympa/default.nix

```diff
@@ -0,0 +1,116 @@
+{ stdenv, perlPackages, fetchurl, makeWrapper
+, dataDir ? "/var/lib/sympa"
+}:
+
+let
+  pname = "sympa";
+  version = "6.2.44";
+in
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/sympa-community/${pname}/releases/download/${version}/${pname}-${version}.tar.gz";
+    sha256 = "10qzzjpl9vkkh3qlw4z44wmgcf8qlnazncl33xv4fv6wdg2wns2v";
+  };
+
+  configureFlags = [
+    "--without-initdir"
+    "--without-unitsdir"
+    "--without-smrshdir"
+
+    "--with-lockdir=/run/lock"
```

Comment:
This directory is owned by root so I doubt sympa will be able to write there. `/run/sympa` would be a better choice imo

Reply:
Or something like `RuntimeDirectory = "lock/sympa"` in the systemd service and then `/run/lock/sympa` here.